libs/transloco/src/lib/transloco.providers.ts

```
@@ -1,9 +1,15 @@
 import {
   EnvironmentProviders,
+  inject,
   makeEnvironmentProviders,
+  provideEnvironmentInitializer,
   Provider,
   Type,
 } from '@angular/core';
+import {
+  TranslocoService,
+  setGlobalTranslateService,
+} from './transloco.service';
```

ESLint

[error] There should be no empty line within import group

(import/order)